ompi/mca/op/aarch64/op_aarch64.h

```diff
@@ -24,6 +24,12 @@
 
 BEGIN_C_DECLS
 
+#if OMPI_MCA_OP_SVE_EXTRA_FLAGS
+#define SVE_ATTR __attribute__ ((__target__ ("+sve")))
+#else
+#define SVE_ATTR
```

Comment:
We tend to prefix our macros so that we know that they were created / set by Open MPI-specific logic.

```suggestion
#define OMPI_SVE_ATTR __attribute__ ((__target__ ("+sve")))
#else
#define OMPI_SVE_ATTR
```

Reply:
Makes sense